.gitmodules

```diff
@@ -19,3 +19,6 @@
 [submodule "gluten"]
 	path = gluten
 	url = https://github.com/anmonteiro/gluten.git
+[submodule "websocketaf"]
```

Comment:
I always wonder about this.
https://stackoverflow.com/questions/5033441/where-does-git-store-the-sha1-of-the-commit-for-a-submodule